oidc-js/src/utils/sign-in.ts

```diff
@@ -69,9 +69,9 @@ import {
     TokenRequestHeader,
     TokenResponseInterface,
     UserInfo,
-    WebWorkerConfigInterface,
-    isWebWorkerConfig
+    WebWorkerConfigInterface
 } from "../models";
+import { isWebWorkerConfig } from "../helpers";
```

Comment:
This import should come before the ../models import

Reply:
Resolved by a0897b4